src/components/common/Extensions/UserHub/partials/TransactionsTab/partials/GroupedTransactionContent.tsx

```diff
@@ -56,16 +56,15 @@ const GroupedTransactionContent: FC<GroupedTransactionContentProps> = ({
     toggleCancelConfirmation,
     handleCancelTransaction,
     canBeSigned,
-  } = useGroupedTransactionContent(
+  } = useGroupedTransactionContent({
     id,
-    error,
```

Comment:
why was `error` removed from here? not used ?

Reply:
Yep! It's unused here